app/components/post_list/post_list.tsx

```diff
@@ -145,6 +149,9 @@ const PostList = ({
     }, []);
 
     const onRefresh = useCallback(async () => {
+        if (disablePullToRefresh) {
+            return;
+        }
```

Comment:
at this point is late, cause the callback is being executed, I do thing that you need to change the `enabled` prop of the `PostListRefreshControl` component in line 375